lib/cinema/ui/pages/main_page/session_card.dart

```diff
@@ -133,24 +151,119 @@ class SessionCard extends HookConsumerWidget {
+                                          image: data.first.image,
+                                          fit: BoxFit.cover),
+                                      boxShadow: [
+                                        BoxShadow(
+                                          color: Colors.grey.withOpacity(0.2),
+                                          spreadRadius: 5,
+                                          blurRadius: 7,
+                                          offset: const Offset(0, 3),
+                                        ),
+                                      ],
                                     ),
-                                  ],
-                                ),
+                                  ),
+                                  if (session.start.isAfter(DateTime.now()))
+                                    Positioned(
+                                        top: 10,
+                                        right: 10,
+                                        child: AnimatedBuilder(
+                                            animation: animation,
+                                            builder: (context, child) {
+                                              return GestureDetector(
+                                                onTap: () {
+                                                  if (animation.isCompleted) {
+                                                    animation.reverse();
+                                                  } else {
+                                                    animation.forward();
+                                                  }
+                                                  cinemaTopicsNotifier
+                                                      .toggleSubscription(
+                                                          session.id);
+                                                  if (selected) {
+                                                    localNotificationService
+                                                        .cancelNotificationById(
+                                                            session.id);
+                                                    displayToast(
+                                                        context,
+                                                        TypeMsg.msg,
+                                                        "Le rappel a été supprimé");
+                                                  } else {
+                                                    localNotificationService.showNotification(
+                                                        Message(
+                                                            actionModule: '',
+                                                            actionTable: '',
+                                                            content:
+                                                                'La séance '
```

Comment:
You should use text constants.

Reply:
This is a first iteration, the text might be updated